health.go

```diff
@@ -280,9 +281,18 @@ func GetSysErrors(ctx context.Context, addr string) SysErrors {
 		se.Errors = append(se.Errors, SysErrAuditEnabled)
 	}
 
+	pkgCmd, arg := getPkgListCmd()
```

Comment:
Use distro independent way to find `updatedb` provided by `mlocate` package in pure Go by
```go
if _, err := exec.LookPath("updatedb"); err == nil {
    return errors.New("updatedb is installed")
}
```